### ISSUE TITLE
Typechecking in docusaurus 2.0.0-beta.8 fails

### ISSUE THREAD

**Issue by TGTGamer** [sjwall/mdx-mermaid#10]
**Describe the bug**
Type checking error - latest versions of docusaurus

**To Reproduce**
Steps to reproduce the behaviour including how the markdown file is processed:

1. install docusaurus latest
2. install mdx-mermaid

**Expected behavior**
No errors

**Screenshots**
![image](https://user-images.githubusercontent.com/11413796/138563227-00a6d69c-280c-49b3-9d02-8ed828f3a75c.png)

**Stack trace**
```
Conversion of type '{ docs: { remarkPlugins: typeof import("d:/Jonathan/Documents/Repositories/devspace/Videndum/projects/Eventiva/node_modules/mdx-mermaid/index")[]; sidebarPath: string; editUrl: string; }; blog: { ...; }; theme: { ...; }; }' to type 'Options' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Types of property 'docs' are incompatible.
    Type '{ remarkPlugins: typeof import("d:/Jonathan/Documents/Repositories/devspace/Videndum/projects/Eventiva/node_modules/mdx-mermaid/index")[]; sidebarPath: string; editUrl: string; }' is not comparable to type 'false | Partial<PluginOptions>'.
      Type '{ remarkPlugins: typeof import("d:/Jonathan/Documents/Repositories/devspace/Videndum/projects/Eventiva/node_modules/mdx-mermaid/index")[]; sidebarPath: string; editUrl: string; }' is not comparable to type 'Partial<PluginOptions>'.
        Types of property 'remarkPlugins' are incompatible.
          Type 'typeof import("d:/Jonathan/Documents/Repositories/devspace/Videndum/projects/Eventiva/node_modules/mdx-mermaid/index")[]' is not comparable to type 'RemarkOrRehypePlugin[]'.
            Type 'typeof import("d:/Jonathan/Documents/Repositories/devspace/Videndum/projects/Eventiva/node_modules/mdx-mermaid/index")' is not comparable to type 'RemarkOrRehypePlugin'.
              Type 'typeof import("d:/Jonathan/Documents/Repositories/devspace/Videndum/projects/Eventiva/node_modules/mdx-mermaid/index")' is not comparable to type '[Function, Record<string, unknown>]'.ts(2352)
```

```js

/** @type {import('@docusaurus/types').Config} */
const config = {
  presets: [
      '@docusaurus/preset-classic',
      /** @type {import('@docusaurus/preset-classic').Options} */
      ({
        docs: {
          remarkPlugins: [require('mdx-mermaid')],
          sidebarPath: require.resolve('./sidebars.js'),
        },
      }),
    ],
  ],
};
```

**Comment by sjwall** [comment]
Thank you for raising this, the fix for this is released in v1.1.1, please let me know if it doesn't work

**Comment by TGTGamer** [comment]
Confirmed
